src/main/java/core/basesyntax/dp/Storage.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.dp;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static Map<Fruit, Integer> storage = new HashMap<>();
+
+    public static void put(Fruit fruit, Integer quantity) {
+        storage.put(fruit, quantity);
+    }
+
+    public static Integer get(Fruit fruit) {
+        return storage.get(fruit);
+    }
+
+    public static Map<Fruit, Integer> getStorage() {
```

Comment:
```suggestion
    public static Map<Fruit, Integer> getAll() {
```
just a more consistent naming